internal/processor/accounts.go

```diff
@@ -64,7 +64,10 @@ func (p *Processor) GetPlatformFilterFromPrimaryAccount(ctx context.Context, pee
 var looksLikeAPIVersionRx = regexp.MustCompile(`^v[0-9][1-9]*$`)
 
 // CreateOrUpdate can be used on an API account and returns the database representation of it.
-func (p *Processor) CreateOrUpdateAccount(ctx context.Context, account keppel.Account, fd keppel.FederationDriver, userInfo audittools.UserInfo, r *http.Request, getSubleaseToken func(models.Peer) (string, *keppel.RegistryV2Error)) (models.Account, *keppel.RegistryV2Error) {
+func (p *Processor) CreateOrUpdateAccount(ctx context.Context, account keppel.FullAccount, fd keppel.FederationDriver, userInfo audittools.UserInfo, r *http.Request, getSubleaseToken func(models.Peer) (string, *keppel.RegistryV2Error)) (models.Account, *keppel.RegistryV2Error) {
```

Comment:
The `fd` argument is not required anymore since Processor gains a FederationDriver slot in this PR.